control-plane/tests/test_phase21_end_to_end_validation.py

```
@@ -0,0 +1,414 @@
+        joined_logs = "\n".join(log_output.output)
+        self.assertIn('"event":"wazuh_ingest_rejected"', joined_logs)
+        self.assertIn('"reason":"missing_bearer_secret"', joined_logs)
+
+    def test_phase21_end_to_end_restore_and_readiness_preserve_phase20_live_path(
+        self,
+    ) -> None:
+        _, service = self._build_service()
+        service, promoted_case, evidence_id, reviewed_at = self._build_phase19_in_scope_case(
+            service=service
+        )
+        approved_request, approval_decision, execution, reconciliation = (
```

Ruff (0.15.9)

[warning] Unpacked variable `approved_request` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)